src/Contributors.tsx

```diff
@@ -2,26 +2,27 @@ import React from 'react';
 import { StyleSheet, Text, ScrollView } from 'react-native';
 
 const Contributors = () => {
-    const contributorsList = ['Melisa Im', 'Monica Lam', 'Linden Chiu'];
+  const contributorsList = ['Melisa Im', 'Monica Lam', 'Linden Chiu'];
```

Comment:
this looks like your new contributor PR is not merged in properly

Reply:
I didn't merge the branch shilpa/addContributor to this branch. It was merged to main and then I deleted that branch. The change is in the main branch though.

Reply:
You have to merge main into this PR branch as described in the README